flixel/FlxG.hx

```diff
@@ -707,6 +709,22 @@ class FlxG
 	{
 		return game._state;
 	}
+	
+	private static inline function get_onMobile():Bool
+	{
+		if (onMobile == null)
```

Comment:
What's the point of this check? Why not make it `@:isVar public static var onMobile(get, never):Bool` and
```haxe
private static inline function get_onMobile():Bool {
   return #if js blah #elseif mobile true #else false #end;
}
```